securedrop_client/api_jobs/updatestar.py

```diff
@@ -4,15 +4,14 @@
 from sdclientapi import API, RequestTimeoutError, ServerConnectionError
 from sqlalchemy.orm.session import Session
 
-from securedrop_client.api_jobs.base import ApiJob
+from securedrop_client.api_jobs.base import SingleObjectApiJob
 
 logger = logging.getLogger(__name__)
 
 
-class UpdateStarJob(ApiJob):
-    def __init__(self, source_uuid: str, is_starred: bool) -> None:
-        super().__init__()
-        self.source_uuid = source_uuid
+class UpdateStarJob(SingleObjectApiJob):
```

Comment:
I don't think `UpdateStarJob` should be a `SingleObjectApiJob` because the user could click on the star twice, for instance, and the second time shouldn't be dropped because the first job is still enqueued.

Reply:
very good point, I'll update this to exclude that job for now